src/sdk/EVMC.ts

```diff
@@ -202,11 +202,11 @@ class EVMC extends BasicSDK {
         });
     };
 
-    callContractMethod = async (methodName: string, params: any[] = []): Promise<any> => {
+    callContractMethod = async (methodName: string, params: any[] = [], callOpts: any[] = []): Promise<any> => {
```

Comment:
[memo]

I Added argument to specify blocknumber
ref
API methods.myMethod.call
https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html#methods-mymethod-call